lib/components/sidebarCard/sidebarCard.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Card as AntCard } from 'antd';
+import styles from './sidebarCard.css';
```

Comment:
Since this is only expected to be used with the sidebar should it live in the `sidebar/` directory?

Reply:
fixed